extras/indexing/src/main/java/mvm/rya/indexing/external/tupleSet/BindingSetStringConverter.java

```diff
@@ -57,48 +57,48 @@
     private static final ValueFactory valueFactory = new ValueFactoryImpl();
 
     @Override
-    public String convert(BindingSet bindingSet, VariableOrder varOrder) {
+    public String convert(final BindingSet bindingSet, final VariableOrder varOrder) {
         checkBindingsSubsetOfVarOrder(bindingSet, varOrder);
 
         // Convert each Binding to a String.
-        List<String> bindingStrings = new ArrayList<>();
-        for(String varName : varOrder) {
+        final List<String> bindingStrings = new ArrayList<>();
+        for(final String varName : varOrder) {
             if(bindingSet.hasBinding(varName)) {
                 // Add a value to the binding set.
                 final Value value = bindingSet.getBinding(varName).getValue();
                 final RyaType ryaValue = RdfToRyaConversions.convertValue(value);
-                String bindingString = ryaValue.getData() + TYPE_DELIM + ryaValue.getDataType();
+                final String bindingString = ryaValue.getData() + TYPE_DELIM + ryaValue.getDataType();
                 bindingStrings.add(bindingString);
             } else {
                 // Add a null value to the binding set.
-                bindingStrings.add(NULL_VALUE_STRING);                
+                bindingStrings.add(NULL_VALUE_STRING);
             }
         }
-        
+
         // Join the bindings using the binding delim.
         return Joiner.on(BINDING_DELIM).join(bindingStrings);
     }
 
     /**
      * Checks to see if the names of all the {@link Binding}s in the {@link BindingSet}
      * are a subset of the variables names in {@link VariableOrder}.
-     * 
+     *
      * @param bindingSet - The binding set whose Bindings will be inspected. (not null)
      * @param varOrder - The names of the bindings that may appear in the BindingSet. (not null)
      * @throws IllegalArgumentException Indicates the names of the bindings are
-     *   not a subset of the variable order. 
+     *   not a subset of the variable order.
      */
-    private static void checkBindingsSubsetOfVarOrder(BindingSet bindingSet, VariableOrder varOrder) throws IllegalArgumentException {
+    protected static void checkBindingsSubsetOfVarOrder(final BindingSet bindingSet, final VariableOrder varOrder) throws IllegalArgumentException {
```

Comment:
You made this projected, but never call it outside of this class. It should remain private.